integration-tests/jest.integration.config.ts

```diff
@@ -0,0 +1,31 @@
+import type { Config } from '@jest/types'
+
+const config: Config.InitialOptions = {
+  coveragePathIgnorePatterns: ['/node_modules/', '/dist/'],
+  globals: {
+    AWS_REGION: 'eu-west-2',
+    STACK_NAME: 'txma-ticf-integration',
+    ZENDESK_ADMIN_EMAIL: 'txma-team2-ticf-admin-dev@test.gov.uk',
+    ZENDESK_AGENT_EMAIL: 'txma-team2-ticf-approver-dev@test.gov.uk',
+    ZENDESK_END_USER_EMAIL: 'txma-team2-ticf-analyst-dev@test.gov.uk',
+    ZENDESK_END_USER_NAME: 'Txma-team2-ticf-analyst-dev'
+  },
+  preset: 'ts-jest',
+  reporters: [
+    'default',
+    [
+      'jest-junit',
```

Comment:
Reasons for JUnit:
- The pipeline requires a `junit.xml` file - which I assume gets uploaded to allure, so not sure that's true regarding the report detail. Although happy to be proved wrong!
- We've been using JUnit this whole time anyway as that was what was defined in the reporter part of the config (it would override whatever jest-allure would be doing)
- The `jest-allure` library hasn't been updated in 3 years i.e. it's not maintained, whereas `jest-junit` was updated 5 days ago
- `jest-allure` would force us to use Jasmine as the test runner. This is a bad thing as it's what Jest was forked from and handles async very differently I believe

Either way we definitely don't need both of these things, and since we've been using JUnit this whole time I suggest we keep that up!

RE output location, jest-junit puts this in the required location for report generation. Now I've updated the readme you can try this out following the instructions. Let me know if they're not clear!

Reply:
I'll try this locally. I think we have been using both, as I have been generating reports from the jest-allure output folder. It was my fault really for forgetting to remove the other, but you have good points supporting sticking to jest-junit. I'll just double check that the report from jest-junit captures the same amount of detail, for the sake of GDS compliance.

Reply:
Update: I've checked with the report from jest-junit. The details are the same, so we are good with this.